File: playground/typescript/src/frontend/GetOrder.ts
import type { Order } from "../domain/entities/Order.js";

/**
 * VIOLATION: Backend use case placed in frontend.
 * Should live in application/use_cases/. Refactoring to enforce
 * frontend/backend boundary would move it and update imports.
 */
export function getOrderHandler(orderId: string): Order | null {
  return null;
}

File: playground/python/frontend/get_order.py
"""Get order handler — VIOLATION: backend use case placed in frontend.

This module should live in application/use_cases/. Refactoring to enforce
frontend/backend boundary would move it and update imports.
"""

from __future__ import annotations

from playground.python.domain.entities.order import Order


def get_order_handler(_order_id: str) -> Order | None:
    """Frontend handler that fetches an order (duplicates get_order use case)."""
    return None  # Stub; should delegate to application/use_cases/get_order
